src/files/BrsFile.ts

```diff
@@ -77,6 +77,24 @@ export class BrsFile {
         this.srcPath = value;
     }
 
+    /**
+     * Should this file be written to disk
```

Comment:
```suggestion
     * Will this file result in only comment or whitespace output? If so, it can be excluded from the output if that bsconfig setting is enabled.
```